src/More/zendeskAPI.ts

```diff
@@ -0,0 +1,75 @@
+import env from "react-native-config"
+
+const defaultHeaders = {
+  "content-type": "application/json",
+}
+
+type EnvironmentProps = {
+  os: string
+  osVersion: string
+  appVersion: string
+}
+
+type FeedbackProps = {
+  subject: string
+  name: string
+  body: string
+  environment: EnvironmentProps
+}
+
+interface NetworkSuccess {
+  kind: "success"
+}
+
+interface NetworkFailure<U> {
+  kind: "failure"
+  error: U
+}
+
+export type FeedbackError = "Unknown" | "ZendeskError"
+
+export type NetworkResponse<U = "Unknown"> = NetworkSuccess | NetworkFailure<U>
```

Comment:
```suggestion
export type NetworkResponse<Error = "Unknown"> = NetworkSuccess | NetworkFailure<Error>
```
or perhaps `T`,  the `U` jumps out as and artifact of a refactor that wasnt fully finished.